interface/src/lib/stores/socket.ts

```
@@ -58,9 +58,13 @@ function createWebSocket() {
 				return;
 			}
 			
-			// 🌙 Validate before destructuring
+			// 🌙 Non-object binary = raw monitor data (not msgpack); route directly and skip destructuring
 			if (!payload || typeof payload !== 'object') {
-				console.error('[WebSocket] Invalid payload:', payload);
+				if (binary) {
+					listeners.get('monitor')?.forEach((listener) => listener(new Uint8Array(message.data)));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)